docs/UserGuide.md

```diff
@@ -165,6 +165,8 @@ Action | Format | Window
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. for the `help` command, if you specify `help 123`, it will be interpreted as `help`.
 
+* For all prefixes (except the first), the user should ensure they are preceded by a space for the parser to recognise it as a new prefix, instead of a parameter in the previous prefix. For example, `editStudent i/e/hello@example.com` parsers the parameter for `i/` as `e/hello@example.com`, and does not detect any input for the `e/` field. On the other hand, for the command `editStudent i/ e/hello@example.com`, `i/` is detected to have no arguments, while `e/` has `hello@example.com` as an argument. As such, the second command will not succeed since the `i/` field is blank. 
```

Comment:
will the first command succeed? 
The last sentence "As such, the second command will not succeed since the `i/` field is blank." might be abit misleading that the 1st command actually succeeds. Perhaps can rephrase to, "ensure that compulsory fields are filled in." or smth like that?

Reply:
You're right, I missed out an index

Reply:
Hmm I think it's clear enough, since I mentioned in the first command that the `i/` field is not parsed as blank.

Reply:
ok!